examples/fastify/package.json

```diff
@@ -36,23 +36,23 @@
     "url": "https://github.com/open-telemetry/opentelemetry-js/issues"
   },
   "dependencies": {
-    "@opentelemetry/api": "^1.0.2",
-    "@opentelemetry/exporter-jaeger": "^1.0.0",
-    "@opentelemetry/exporter-zipkin": "^1.0.0",
-    "@opentelemetry/exporter-collector": "^0.25.0",
-    "@opentelemetry/instrumentation": "^0.27.0",
-    "@opentelemetry/instrumentation-fastify": "^0.25.0",
-    "@opentelemetry/instrumentation-http": "^0.26.0",
-    "@opentelemetry/sdk-trace-node": "^1.0.0",
-    "@opentelemetry/resources": "^1.0.0",
-    "@opentelemetry/semantic-conventions": "^1.0.0",
-    "@opentelemetry/sdk-trace-base": "^1.0.0",
-    "axios": "^0.21.1",
+    "@opentelemetry/api": "1.7.0",
+    "@opentelemetry/sdk-node": "^0.46.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "^0.46.0",
+    "@opentelemetry/exporter-metrics-otlp-proto": "^0.46.0",
+    "@opentelemetry/instrumentation": "^0.46.0",
+    "@opentelemetry/instrumentation-http": "^0.46.0",
+    "@opentelemetry/instrumentation-fastify": "^0.32.6",
+    "@opentelemetry/sdk-trace-node": "~1.19.0",
+    "@opentelemetry/resources": "~1.19.0",
+    "@opentelemetry/semantic-conventions": "~1.19.0",
+    "@opentelemetry/sdk-trace-base": "~1.19.0",
```

Comment:
The tilde ranges were intentional, right? E.g. this limits `@opentelemetry/sdk-trace-base` to `>=1.19.0 <1.20.0`.

Reply:
They were, yes. This is to prevent any problems with mixing experimental and stable packages. The example is not part of the workspace, so it should not hold any packages back for the rest of the repo :slightly_smiling_face: